MapboxNavigation/NavigationViewController.swift

```diff
@@ -687,6 +688,17 @@ extension NavigationViewController: NavigationServiceDelegate {
         return navigationComponents.allSatisfy { $0.navigationServiceShouldDisableBatteryMonitoring(service) }
     }
     
+    public func locationManagerDidChangeAuthorization(_ manager: CLLocationManager) {
+        let title = NSLocalizedString("ENABLE_PRECISE_LOCATION", bundle: .mapboxNavigation, value: "Enable precise location to  navigate", comment: "Label indicating precise location is off and needs to be turned on to navigate")
+        if #available(iOS 14.0, *), manager.accuracyAuthorization == .reducedAccuracy {
+            // It takes about 9 seconds to recognize the banner and change location authorization in settings
+            showStatus(title: title, spinner: false, duration: 9, animated: true, interactive: false)
```

Comment:
Unlike some of the other things we use the status view for, approximate location mode is persistent, requiring the user to take action to disable it. Nine seconds seems like a long time when staring at a simulator, but if you’re a driver juggling multiple tasks, you might still miss the status. As tail work or perhaps as part of this PR, consider leaving the status visible. Apparently if the duration is infinite, the status won’t autohide:

https://github.com/mapbox/mapbox-navigation-ios/blob/e8abb39222563efc56d4721bb25a0167fba95091/MapboxNavigation/StatusView.swift#L122

There’s a separate method on StatusView to manually hide a status, though it isn’t currently exposed as part of the NavigationStatusPresenter protocol.

Reply:
I noticed this. I tried to set a longer duration, but it wouldn't go away if locationAuthorization was changed. I will look into exposing that part of the protocol!